dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -482,23 +482,13 @@
      * simple the registry for provider.
      * @since 2.7.0
      */
-    public static final String SIMPLE_PROVIDER_CONFIG_KEY = "simple.provider.config";
-    /**
-     * simple the registry for consumer.
-     * @since 2.7.0
-     */
-    public static final String SIMPLE_CONSUMER_CONFIG_KEY = "simple.consumer.config";
+    public static final String SIMPLIFIED_KEY = "simplified";
```

Comment:
in the future, we should categorize such constants into groups, for this case, it'd be better and more clear if we move it into RegistryConstants. @cvictory, would you mind to file one issue against this suggestion?